e2e/internal/client/cli.go

```diff
@@ -214,6 +214,33 @@ type PackageInput struct {
 	Tag     string
 }
 
+// PipelineInitInput contains the parameters for calling copilot pipeline init.
+type PipelineInitInput struct {
+	AppName      string
```

Comment:
nit: If we don't need `AppName` then we can remove them from the structs for now!